File: Solution/Backend/dotnet/DataHelperClasslib_Generic/CSVFileIo/Entities/Candidate.cs
namespace CSVFileIO.Entity
{
    public class Candidate
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Email { get; set; }

        public string AppliedPosition { get; set; }

        public int SkillsMatch { get; set; }

        public string ExperienceLevel { get; set; }

        public int AssessmentScore { get; set; }

        public string InterviewStatus { get; set; }

        public DateTime ApplicationDate { get; set; }


        public override string ToString()
        {
            return Id + "," +
                   Name + "," +
                   Email + "," +
                   AppliedPosition + "," +
                   SkillsMatch + "," +
                   ExperienceLevel + "," +
                   AssessmentScore + "," +
                   InterviewStatus + "," +
                   ApplicationDate.ToString("yyyy-MM-dd");
        }
    }

    
}

File: Solution/Backend/java/webapi/src/main/java/com/transflower/tflcomentor/fileio/Entity/Candidate.java
package com.transflower.tflcomentor.fileio.Entity;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder

public class Candidate {

    private int id;
    private String name;
    private String email;
    private String appliedPosition;
    private int skillsMatch;
    private String experienceLevel;
    private int assessmentScore;
    private String interviewStatus;
    private String applicationDate;
}